### ISSUE TITLE
How to call `mobile-app/*` endpoints?

### ISSUE THREAD

**Issue by mukundt** [timdorr/tesla-api#714]
I'm trying to access the [Self Diagnostics](https://electrek.co/2022/07/18/tesla-new-self-diagnostic-feature-app/) feature, which I [believe](https://twitter.com/tesla_adri/status/1548295084524707840) falls under `mobile-app/macgyver/urgent-autodiag-result`. But calling `api('SERVICE_MACGYVER_DIAGNOSTIC_RESULT')` returns a 404. Do I need to treat `mobile-app/` endpoints differentely?

**Comment by core-hacked** [comment]
Yes, these either fall under different urls or require different authentication / token generation.

**Comment by norenz92** [comment]
Anyone actually know how to fetch these endpoints?
